README.md

```diff
@@ -198,10 +199,13 @@ const fullOrPartialPages = await notion.databases.query({
   database_id: "897e5a76-ae52-4b48-9fdf-e71f5945d1af",
 })
 for (const page of fullOrPartialPages.results) {
-  if (!isFullPage(page)) {
+  if (!isFullPageOrDatabase(page)) {
     continue
   }
-  // The page variable has been narrowed from PageObjectResponse | PartialPageObjectResponse to PageObjectResponse.
+  // The page variable has been narrowed from
+  //      PageObjectResponse | PartialPageObjectResponse | DatabaseObjectResponse | PartialDatabaseObjectResponse
+  // to
+  //      PageObjectResponse | DatabaseObjectResponse.
```

Comment:
I wonder if we should show / have our helpers be better at narrowing further. I think this is a good start though.